homeassistant/components/fritzbox/sensor.py

```diff
@@ -33,16 +33,12 @@ async def async_setup_entry(
     coordinator = hass.data[FRITZBOX_DOMAIN][entry.entry_id][CONF_COORDINATOR]
 
     for ain, device in coordinator.data.items():
-        if (
-            device.has_temperature_sensor
-            and not device.has_switch
-            and not device.has_thermostat
-        ):
+        if device.has_temperature_sensor and not device.has_thermostat:
             entities.append(
                 FritzBoxTempSensor(
                     {
-                        ATTR_NAME: f"{device.name}",
-                        ATTR_ENTITY_ID: f"{device.ain}",
+                        ATTR_NAME: f"{device.name} Temperature",
+                        ATTR_ENTITY_ID: f"{device.ain}_temperature",
```

Comment:
This is not changing the entity_id, but this is used as an unique_id:

```py
# __init__.py
self._unique_id = entity_info[ATTR_ENTITY_ID]
```

Thus, this doesn't have to be a breaking change, as the unique ID can be migrated in the entity registry on the integration setup.

Reply:
many thanks for this hint :slightly_smiling_face:
but since this PR is stale closed, I have opened a new one #52558 which also includes this suggestion and those is not a breaking change anymore.